tests/integration/bundles/kfp_1.7_stable_install.yaml.j2

```diff
@@ -87,6 +87,7 @@ applications:
     charm: {{ kfp_schedwf }}
     resources: {{ kfp_schedwf_resources }}
     scale: 1
+    trust: true
```

Comment:
You should add `trust: true` in two places in this file. See https://github.com/canonical/kfp-operators/blob/6acdcf3ed17f5c99b208afa4380ef6454490d894/tests/integration/bundles/kfp_1.7_stable_install.yaml.j2#L57-L60

Reply:
+1